### ISSUE TITLE
Farengar items and other vendor chests(?) improperly looted

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#31]
user report from Nexus: https://forums.nexusmods.com/index.php?showtopic=8799893/#entry81978373

> If you have the loot radius to high it'll loot the merchant chests. I got all spellbooks farengar sells.

**Comment by SteveTownsend** [comment]
This seems to be a problem with multiple vendor chests. Hopefully can auto-detect more of them with new rules at load time.

**Comment by SteveTownsend** [comment]
Chests that are not owned or locked are fair game. Added logic to detect and block VendorGold LVLI containers

**Comment by SteveTownsend** [comment]
Test evidence - debug output from visit to Farengar after data load:
```
J:\GitHub\SmartHarvestSE\dataCase.cpp(431): [MESSAGE] {34920} (2020-06-16 05:33:18 PM) /0x0010f9b6 in Container Chest/0x000a298a not VendorGold
J:\GitHub\SmartHarvestSE\dataCase.cpp(431): [MESSAGE] {34920} (2020-06-16 05:33:18 PM) /0x0010c1cc in Container Chest/0x000a298a not VendorGold
J:\GitHub\SmartHarvestSE\dataCase.cpp(431): [MESSAGE] {34920} (2020-06-16 05:33:18 PM) /0x00105241 in Container Chest/0x000a298a not VendorGold
J:\GitHub\SmartHarvestSE\dataCase.cpp(431): [MESSAGE] {34920} (2020-06-16 05:33:18 PM) /0x00105240 in Container Chest/0x000a298a not VendorGold
J:\GitHub\SmartHarvestSE\dataCase.cpp(431): [MESSAGE] {34920} (2020-06-16 05:33:18 PM) /0x000a298e in Container Chest/0x000a298a not VendorGold
J:\GitHub\SmartHarvestSE\dataCase.cpp(431): [MESSAGE] {34920} (2020-06-16 05:33:18 PM) /0x0010523f in Container Chest/0x000a298a not VendorGold
J:\GitHub\SmartHarvestSE\dataCase.cpp(423): [MESSAGE] {34920} (2020-06-16 05:33:18 PM) Block Vendor Container Chest/0x000a298a
J:\GitHub\SmartHarvestSE\tasks.cpp(1091): [VERBOSE] {34184} (2020-06-16 05:34:46 PM) Process REFR 0x000abb42 with base object Chest/0x000a298a
J:\GitHub\SmartHarvestSE\debugs.cpp(153): [MESSAGE] {34184} (2020-06-16 05:34:46 PM) 000a298a 1c(28) [Chest]
J:\GitHub\SmartHarvestSE\tasks.cpp(112): [MESSAGE] {34184} (2020-06-16 05:34:46 PM) Block owned/illegal-to-loot REFR: Chest/0x000a298a
```

**Comment by ab9rf** [comment]
I just had this happen on 1.7.1.0 with Farengar. Problem may not be solved.

**Comment by SteveTownsend** [comment]
The vendor chest logic is not specific to Farengar's chest. If is was not working universally there would be more reports, I think.

**Comment by SteveTownsend** [comment]
Hmm it looks like this might have gotten broken, testing now

**Comment by SteveTownsend** [comment]
I strongly suspect this is broken, as you pointed out - not sure when, but
a lot of change in the recent releases. I'll fix in 1.8.0.0, anyway

**Comment by SteveTownsend** [comment]
Test evidence - blocked at startup:
```
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(473): [MESSAGE] {38712} (2020-07-14 02:19:02 PM) Block Vendor Container Chest/0x000a298a
```
Scanned but not looted when in range, whitelist Dragonsreach
```
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1102): [VERBOSE] {3736} (2020-07-14 02:27:01 PM) Process REFR 0x000abb42 with base object Chest/0x000a298a
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(479): [MESSAGE] {3736} (2020-07-14 02:27:01 PM) scanning container/body Chest/0x000abb42
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(174): [MESSAGE] {3736} (2020-07-14 02:27:01 PM) CONT 000a298a 1c(28) [Chest]
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(228): [VERBOSE] {3736} (2020-07-14 02:27:01 PM) Check REFR 0x000abb42 to not-locked container Chest/0x000a298a
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(551): [VERBOSE] {3736} (2020-07-14 02:27:01 PM) glow container with enchanted object Chest/0x000abb42
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(562): [VERBOSE] {3736} (2020-07-14 02:27:01 PM) glow container with valuable object Chest/0x000abb42
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(817): [VERBOSE] {3736} (2020-07-14 02:27:01 PM) Trigger glow for Chest/0x000abb42
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1102): [VERBOSE] {3736} (2020-07-14 02:27:01 PM) Process REFR 0x000e27bf with base object Silver Goblet/0x00098621
```

**Comment by SteveTownsend** [comment]
Re-fixed in v1.8.0.0

**Comment by ab9rf** [comment]
Confirm that this appears to be fixed in 1.9.0.0. Thanks for your prompt attention to this.

**Comment by SteveTownsend** [comment]
Thanks for letting me know
